common/scala/src/main/scala/whisk/core/entity/Exec.scala

```diff
@@ -93,9 +93,12 @@ protected[core] case class BlackBoxExec(image: String) extends Exec(Exec.BLACKBO
     override val sentinelledLogs = false
 }
 
-protected[core] case class SequenceExec(code: String, components: Vector[String]) extends Exec(Exec.SEQUENCE) {
+/**
+ * add temporary field that holds the "fixed" names for components where the '_' is replaced by the user's namespace
+ */
+protected[core] case class SequenceExec(code: String, components: Vector[FullyQualifiedEntityName]) extends Exec(Exec.SEQUENCE) {
     val image = Exec.imagename(Exec.NODEJS)
-    def size = components.map(_ sizeInBytes).reduce(_ + _)
+    def size = ByteSize.fromString("0B") // not used for the hacky implementation and not used for the new implementation either
```

Comment:
Not following the comment? If this is supposed to stay in, use `0.bytes` instead.